app/src/main/java/ru/practicum/android/diploma/details/presentation/state/DetailState.kt

```diff
@@ -4,6 +4,10 @@ import ru.practicum.android.diploma.details.domain.models.ProfessionDetail
 
 sealed class DetailState {
```

Comment:
🍏 Так как DetailState не хранит какого-то состояния, можно объявить его интерфейсом:

```suggestion
sealed interface DetailState {
```